src/DotNetProjectFile.Analyzers/Licensing/Licenses.cs

```diff
@@ -93,7 +107,18 @@ public static LicenseExpression FromExpression(string? licenseExpression)
 
     public static LicenseExpression FromUrl(string? licenseUrl)
```

Comment:
It makes more sense to define this on LicenseExpression itself I guess.

Reply:
Not so sure about that. If we want to refactor it, I would like to do it in a different PR.